app/controllers/api/v1/users_controller.rb

```diff
@@ -40,6 +41,12 @@ def destroy
     render json: @users
   end
 
+  # GET /users/1/appointments
+  def appointments
+    @appointments = current_user.appointments
+    render json: @appointments
+  end
+
```

Comment:
Great job :clap:  working on this endpoint, this will help user see their appointments from the front end